test/unit/model/qti/ItemTest.php

```diff
@@ -21,89 +21,80 @@
 
-        if (!defined('PRODUCT_NAME')) {
-            define('PRODUCT_NAME', self::PRODUCT_NAME);
-        }
-        if (!defined('ROOT_URL')) {
-            define('ROOT_URL', __DIR__ . '/../../../../../');
-        }
-        if (!defined('CONFIG_PATH')) {
-            define('CONFIG_PATH', ROOT_URL . 'config/');
-        }
-        if (!defined('EXTENSION_PATH')) {
-            define('EXTENSION_PATH', ROOT_URL);
-        }
-
-        ServiceManager::getServiceManager()->overload(
-            ApplicationService::SERVICE_ID,
-            $this->createMock(ApplicationService::class)
-        );
-    }
+        define('PRODUCT_NAME', self::PRODUCT_NAME);
 
-    public function testToQTI(): void
-    {
-        $expectedItemQti = <<<ITEM_QTI
-<?xml version="1.0" encoding="UTF-8"?><assessmentItem
-        xsi:schemaLocation=""
-     identifier="Item_1" title="" label="" adaptive="false" timeDependent="false" toolName="TAO">
+        $commonExtensionMock = $this->createMock(common_ext_Extension::class);
+        $commonExtensionMock
+            ->method('getDir')
+            ->willReturn(ROOT_PATH . DIRECTORY_SEPARATOR . 'taoQtiItem' . DIRECTORY_SEPARATOR);
 
-    
-    
-    
-    <itemBody >
-	    </itemBody>
+        $extensionsManagerMock = $this->createMock(common_ext_ExtensionsManager::class);
+        $extensionsManagerMock
+            ->method('getExtensionById')
+            ->willReturn($commonExtensionMock);
 
-    
-    
-    </assessmentItem>
+        $applicationServiceMock = $this->createMock(ApplicationService::class);
 
-ITEM_QTI;
+        $sm = $this->getServiceManagerMock([
+            ApplicationService::SERVICE_ID => $applicationServiceMock,
+            common_ext_ExtensionsManager::class => $extensionsManagerMock
+        ]);
 
+        ServiceManager::setServiceManager($sm);
+    }
+
+    /**
+     * Testing toQTI() method on Item class
+     * @return void
+     */
+    public function testToQTI(): void
+    {
+        $expectedItemQti = file_get_contents(
+            dirname(__DIR__, 2) . '/samples/model/qti/item/testToQti_expectedItemQti.xml'
+        );
 
         $item = new Item();
         $itemQti = $this->removeToolVersionAttribute($item->toQTI());
 
         self::assertEquals($expectedItemQti, $itemQti);
     }
 
+    /**
+     * Testing toQTI() method on Item class with attribute dir=rtl
+     * @return void
+     * @throws \oat\taoQtiItem\model\qti\exception\QtiModelException
+     */
     public function testToQTIWithDirAttributeInItemBody(): void
     {
-        $expectedItemQti = <<<ITEM_QTI
-<?xml version="1.0" encoding="UTF-8"?><assessmentItem
-        xsi:schemaLocation=""
-     identifier="Item_1" title="" label="" adaptive="false" timeDependent="false" toolName="TAO">
-
-    
-    
-    
-    <itemBody dir="rtl">
-	    </itemBody>
-
-    
-    
-    </assessmentItem>
-
-ITEM_QTI;
-
+        $expectedItemQti = file_get_contents(
+            dirname(
+                __DIR__,
```

Comment:
please don't use native php functions ,  rather use our abstractions

Reply:
Can you suggest me something to load file? Other tests use Documetnt class or other xml parsers but I just want to load file contents to use it in assertion.

Reply:
I think it is quite acceptable to use native function for reading some sample data.

Reply:
I already have version using flysystem so I can commit it or scrap it and leave native functions IDK.